tests/modules/withconstructor.py

```diff
@@ -33,9 +35,10 @@ def __init__(self, x: int, y: str, unit: str, u: float, z: List[int]):
             self.x = x
             self.y = y
         # multiple assignments
-        self.x_unit = self.y_unit = z_unit = unit
+        self.x_unit = self.y_unit = unit
         # type of self.z should be undefined, z has been overridden without type annotation
         z, (r, t) = str(x) + y, ('r', 't')
+        print(r, t)
```

Comment:
Is this `print` statement intended?

Reply:
probably not, but this code never gets executed. I will remove this statement in a post-review commit to come :pray: 

TODO